tests/test_nodes.py

```diff
@@ -101,15 +105,26 @@ def test_valid_inputs(type_: type, input_: Any, output_: Any) -> None:
         (FloatNode, ListConfig([1, 2])),
         (FloatNode, {"foo": "var"}),
         (FloatNode, DictConfig({"foo": "var"})),
+        (AnyNode, [1, 2]),
+        (AnyNode, ListConfig([1, 2])),
+        (AnyNode, {"foo": "var"}),
+        (AnyNode, DictConfig({"foo": "var"})),
+        (AnyNode, Person()),
     ],
 )
 def test_invalid_inputs(type_: type, input_: Any) -> None:
     empty_node = type_()
-    with pytest.raises(ValidationError):
-        empty_node._set_value(input_)
 
-    with pytest.raises(ValidationError):
-        type_(input_)
+    if type_ == AnyNode:
+        with pytest.raises(UnsupportedValueType):
```

Comment:
didn't we say we can make UnsupportedValueType a subclass of ValidationError?

Reply:
Ah I thought you meant we could make them on 2.1

Reply:
I misunderstood you, I'll change it.

Reply:
You don't need to special-case AnyNode now that the Exception is a ValidationError.